typescript/web/src/@types/svg.d.ts

```diff
@@ -0,0 +1,18 @@
+import { FC, SVGAttributes } from "react";
+
+declare global {
+  type SvgComponent = FC<SVGAttributes<SVGElement>>;
+}
+
+// FIXME Make this code block actually override NextJS definitions.
```

Comment:
As explained below, the goal is to avoid the casts such as the ones done in `typescript/web/src/components/core/icons/svg/index-svg.ts`. They are only needed because NextJS overrides the types of imported SVG by `any`.
They are alternatives but they involve maintaining our own NextJS global types files or messing hard with the hacky tsconfig we use in our repo.